src/glauth.py

```diff
@@ -32,7 +35,17 @@ def _snap():
     return cache["glauth"]
 
 
-def install():
+def create_default_config(api_port: int, ldap_port: int) -> None:
+    """Create default config with no users."""
+    with open("templates/glauth.toml.j2", "r") as f:
+        template = Template(f.read())
+
+    rendered = template.render(api_port=api_port, ldap_port=ldap_port)
+    with open("/var/snap/glauth/common/etc/glauth/glauth.d/glauth.cfg", "w") as f:
+        f.write(rendered)
```

Comment:
Why not this instead?

```suggestion
    pathlib.Path("/var/snap/glauth/common/etc/glauth/glauth.d/glauth.cfg").write_text(rendered)
```